Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -1829,7 +1829,7 @@ static ErrorOr<LfGlobal> read_lf_global(LittleEndianInputBitStream& stream,
 
 /// G.2 - LfGroup
 static ErrorOr<void> read_lf_group(LittleEndianInputBitStream&,
-    Image& image,
+    Span<Channel> channels,
     FrameHeader const& frame_header)
```

Comment:
"Reading LF group is" -> "Reading an LF group is" or "Reading LF groups is"

Reply:
Fixed!